conf/apt/apt.conf.d/00-kano-default-release

```diff
@@ -0,0 +1 @@
+APT::Default-Release "release";
```

Comment:
Is this how we can switch between different sources now?

Reply:
Yeah, you need to do two things:

 - Create a new file in something like  `/etc/apt/apt.conf.d/99-release` (anything with a higher number than `00-*`) to say `APT::Default-Release "rc";`, for example
 - Create a new file in something like  `/etc/apt/preferences.d/99-pinning` (anything with a higher number than `00-*`) to unpin the other repo, for example:
 
```
Package: *
Pin: release n=rc
Pin-Priority: 500
```

These two steps are handled by `kano-dev`.

In addition, there is the really nice thing that you can directly install a package from `scratch` say without changing sources with:

```
apt-get install -t scratch my-package
```